specification/turn/src/main/java/org/kaazing/specification/turn/internal/Functions.java

```diff
@@ -67,6 +70,34 @@
     }
 
     @Function
+    public static byte[] ipV6XOR(String ip, byte[] transactionId) {
+        InetAddress address;
+        try {
+            address = InetAddress.getByName(ip);
+        } catch (UnknownHostException e) {
+            e.printStackTrace(); // TODO replace with log if available
```

Comment:
If you throw the exception (maybe Runtime), then the k3po runtime should throw the exception and print the stack trace... thoughts?